internal/linker/test/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+semver@1.0.0:
```

Comment:
this file is so short, it makes me wonder (in a separate PR) if we could make it work to say

```
http_file(
   name="npm_semver", urls = ["https://registry.yarnpkg.com/semver/-/semver-1.0.0.tgz#11f18a0c08ed21c988fc2b0257f1951969816615"]
)
external_npm_package(
    name = "npm__semver",
    tars = "that one ☝🏻 "
)
```
or sth so users could opt-out of using npm or yarn and use only Bazel. Probably an experiment we could do along with the spike I've been working on.